report/unsafe/sections/martin_evaluation_challenge.tex

```diff
@@ -0,0 +1,14 @@
+\subsection{Engineering challenge}
+Performance of the robot during navigation on the setup shown in Fig. \ref{fig:factory_setup} was evaluated with the following criterias
+
+\begin{itemize}
+    \item Autonomy of the robot
+    \item Speed and navigation
+    \item Pickup / drop-off performance
+\end{itemize}
+
+The test were performedd at Luleå University of Technology at EISLAB with a local Arrowhead cloud and private network. The robot was able to navigate the factory floor and perform the action but moved slowly. This is because the Dynamixel motors used to translate the robot are slow. Communicating with Arrowhead was working and the test was proved successfull. A video with the whole run can be seen here: https://www.youtube.com/watch?v=YoAFspEC2no. 
```

Comment:
"are slow" -> "can only achieve a low rpm"
slow sounds so negative :)
(and might be to general)

Add hyperlink and (maybe) use shortened URL: https://youtu.be/YoAFspEC2no